src/bokeh/palettes.py

```diff
@@ -1885,6 +1925,17 @@ def gray(n: int) -> Palette:
 # Dev API
 #-----------------------------------------------------------------------------
 
+def _to_rgba_array(palette: Palette) -> npt.NDArray[np.uint8]:
```

Comment:
I've implemented this a separate function that might be useful elsewhere. I wasn't sure if it should be in Dev or Private API sections.

Reply:
If the intention for this function is to be a part of the (dev) API, then it's the right place, though in this case `_` prefix shouldn't be used.

Reply:
Right, I will remove the underscore.